modules/cert-manager-override-arguments.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-arguments_{context}"]
+= Customizing cert-manager by overriding arguments from the cert-manager Operator API
+
+You can override the supported arguments for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+. Add a `spec.controllerConfig` section with the following override arguments:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  ...
+spec:
+  ...
+  controllerConfig: <1>
+    overrideArgs:
+      - '--dns01-recursive-nameservers=<host>:<port>' <2>
+      - '--dns01-recursive-nameservers-only' <3>
+      - '--acme-http01-solver-nameservers=<host>:<port>' <4>
+      - '--v=<verbosity_level>' <5>
+      - '--metrics-listen-address=<host>:<port>' <6>
+  webhookConfig:
+    overrideArgs:
+      - '--v=4'
```

Comment:
Are these values supposed to be fixed? If no, then it can be changed to something like callout <5>